sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/client/TokenExchange.java

```diff
@@ -65,17 +70,22 @@ public static String buildAppCenterGetDbTokenBodyPayload(String partition) {
     public static synchronized ServiceCall getDbToken(
             final String partition,
             HttpClient httpClient,
+            Context context,
             String apiUrl,
             final String appSecret,
             TokenExchangeServiceCallback serviceCallback) {
         AppCenterLog.debug(LOG_TAG, "Getting a resource token from App Center...");
         String url = apiUrl + GET_TOKEN_PATH_FORMAT;
+        final String userToken = TokenStorageFactory.getTokenStorage(context).getToken();
```

Comment:
This context is get when the Storage service initialized, is it using the same storage as identity module?

Reply:
This pattern is the right one to use.

Reply:
You need to use `AuthTokenContext` instead.